app/jobs/order_cycle_opened_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Trigger jobs for any order cycles that recently opened
+class OrderCycleOpenedJob < ApplicationJob
+  def perform
+    ActiveRecord::Base.transaction do
+      recently_opened_order_cycles.find_each do |order_cycle|
+        SubscriptionPlacementJob.perform_later(order_cycle.id)
```

Comment:
I thought that it would be good to have one place where we define the logic selecting and processing order cycles as "opened".
It does seem strange to be creating so many jobs, yes, but it helps "separate concerns" in other ways I guess. I thought it would be more robust this way.

Reply:
I understand but I think this field name is very misleading. What is a notification? I'd use the same terminology everywhere so that code readers will know what this means. It's a webhook. It could be called opened_webhook_sent_at.

I still think it would be much better and very easy to implement this outside the OC table (there's already lots of complexity in OCs).
A table where webhooks calls are registered is important imho. This would be one of them.

Reply:
I think I agree, but this would be better considered as part of https://github.com/openfoodfoundation/openfoodnetwork/issues/10457. So for now, I plan to rename this column to  match specifically what it refers to.